sanic/request.py

```diff
@@ -61,7 +62,10 @@ def json(self):
             try:
                 self.parsed_json = json_loads(self.body)
             except Exception:
-                pass
+                return HTTPResponse(json_dumps(self.body),
```

Comment:
I don't think it makes sense to return an HTTPResponse from a function in the request. What do you think @channelcat?